File: JavaFitNesse/trunk/NCP_FitNesse/src/test/java/com/sqs/nochangeparking/unittests/CheckSayHello.java
package com.sqs.nochangeparking.unittests;

import junit.framework.*;
import com.sqs.nochangeparking.core.*;

public class CheckSayHello extends TestCase{
	
	public void testSayHelloReturnsChuck() {
		assertEquals("Hello Chuck", GreetingServer.sayHello("Chuck"));
	}
	
	public void testSayHelloWithNoNameReturnsHelloStranger() {
		assertEquals("Hello Stranger", GreetingServer.sayHello(""));
	}

}

File: trunk/NoChangeParking/src/NoChangeParking.FitFixtures/CheckSayHello.cs
using System;
using fit;

namespace NoChangeParking.FitFixtures
{
	public class CheckSayHello: ColumnFixture
	{
		public string name;
		NoChangeParking.Core.GreetingServer checkSayHello = new NoChangeParking.Core.GreetingServer();
		
		public string sayHello(){
			return checkSayHello.SayHello(name);
		}
	}
}
